Commands/Reformat Document.tmCommand

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>beforeRunningCommand</key>
+	<string>saveActiveFile</string>
+	<key>command</key>
+	<string>#!/bin/bash
+
+[[ -f "$HOME/.rvm/scripts/rvm" ]] &amp;&amp; . "$HOME/.rvm/scripts/rvm"
+
+(
+bundler_dir="${TM_PROJECT_DIRECTORY:-$(dirname "$TM_FILEPATH")}"
+if [[ -f "$bundler_dir/Gemfile" ]]; then
+  cd "$bundler_dir" &amp;&amp; bundle list rubocop
+  if [[ $? == 0 ]]; then bundler_rubocop='bundle exec rubocop'; fi
+fi
+
+if [[ ! `which rubocop` &amp;&amp; -z "$bundler_rubocop" ]]; then
+  "$DIALOG" tooltip --text "This command requires RuboCop.
+Please install RuboCop via “gem install rubocop”."
+  exit
+fi
+
+aha_found=`which aha`
+if [[ $aha_found ]]; then
+  color='n';
+  format='--html'
+else
+  format='--text'
+fi
+
+rubocop="${TM_RUBOCOP:-"${bundler_rubocop:-rubocop}"}"
```

Comment:
> Not sure how this line works. Is `TM_RUBOCOP` or `bundler_rubocop` preferred?

If `TM_RUBOCOP` is not set, then the command tries to set the variable `rubococp` to the value of `bundler_rubocop`. If `bundler_rubocop` is also not set, then we use the value `rubocop`. This means the current order of preference is:

1. `TM_RUBOCOP`
2. The version of rubocop specified via Bundler
3. The command `rubocop`